src/components/CityWeatherShortcut/CityWeatherShortcut.tsx

```diff
@@ -0,0 +1,39 @@
+import { weatherDataInCitySuccessResponse } from 'mocks/mockData';
+import { Container, Icon, Temperature, Value, TemperatureSign, City } from './CityWeatherShortcut.styled';
+import { CityWeatherShortcutTestIds } from './CityWeatherShortcutTestIds';
+import { CityWeatherShortcutLoader } from './components/CityWeatherShortcutLoader';
+
+type Props = {
+  cityName: string;
+};
+
+const data = weatherDataInCitySuccessResponse;
```

Comment:
**suggestion** move it to the body of the component. 

In a previous comment, I suggested you make those temp variables as close to the final result as we can. they should be moved to the component :)

Reply:
@kamil-borecki-cc done